hera_pspec/uvpspec.py

```diff
@@ -1497,28 +1503,35 @@ def combine_uvpspec(uvps, verbose=True):
     Nblpairts = len(new_blpts)
     Npols = len(new_pols)
 
+
+    #store covariance only if all uvps have stored covariance.
+    u.store_cov = np.all(np.array([uvp.store_cov for uvp in uvps]))
```

Comment:
If we get rid of `uvp.store_cov`, this line can just be
```
store_cov = np.all([hasattr(uvp, 'cov_array') for uvp in uvps])
```

Reply:
changed this.